queries.sql

```diff
@@ -8,3 +8,84 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+/*The BEGIN statement starts a new transaction. The UPDATE statement updates all rows in the 'animals' table
+by setting the 'species' column to 'unspecified'. The SELECT statement verifies that the change was made
+by selecting all rows where the 'species' column is 'unspecified'. The ROLLBACK statement rolls back the transaction,
+undoing the changes made by the UPDATE statement. The final SELECT statement verifies that the 'species' column
+went back to its original state by selecting all rows where the 'species' column is not 'unspecified'.*/
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals; -- verify the change
+ROLLBACK;
+
+SELECT * FROM animals; --verify the change
+/*The BEGIN statement starts a new transaction. The first UPDATE statement updates all rows in the 'animals'
+table where the 'name' column ends with 'mon' by setting the 'species' column to 'digimon'.
+The second UPDATE statement updates all rows in the 'animals' table where the 'species' column is
+NULL (i.e., not already set) by setting the 'species' column to 'pokemon'.
+The SELECT statement verifies that the changes were made by selecting all rows where the 'species'
+column is either 'digimon' or 'pokemon'. The COMMIT statement commits the transaction, making the
+changes permanent. Finally, the last SELECT statement verifies that the changes persist by selecting
+all rows where the 'species' column is either 'digimon' or 'pokemon'.*/
```

Comment:
- [ ] 😄 Kindly read the feedback above for clarity 😄 on comments best practices 👍